app/server-routes/src/main/scala/org/bitcoins/server/routes/CommonRoutes.scala

```diff
@@ -0,0 +1,17 @@
+package org.bitcoins.server.routes
+
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.directives.RouteDirectives
+import org.bitcoins.core.util.EnvUtil
+
+case class CommonRoutes() extends ServerRoute {
+
+  override def handleCommand: PartialFunction[ServerCommand, Route] = {
+    case ServerCommand("getversion", _) =>
+      RouteDirectives.complete {
+        val vec = Vector(("version", ujson.Str(EnvUtil.getVersion)))
+        val obj = ujson.Obj.from(vec)
+        Server.httpSuccess(obj)
+      }
```

Comment:
Is there a reason this can't this be in core routes?

Reply:
is it ok to extend `CoreRoutes` into the `oracleServer`?

Reply:
Oh I see now, I guess it could, but I think common routes makes more sense.